src/NHibernate.Test/Component/Basic/ComponentWithUniqueConstraintTests.cs

```diff
@@ -21,8 +21,8 @@ protected override HbmMapping GetMappings()
 
 			mapper.Component<Person>(comp =>
 			{
-				comp.Property(p => p.Name);
-				comp.Property(p => p.Dob);
+				comp.Property(p => p.Name, m => m.NotNullable(true));
+				comp.Property(p => p.Dob, m => m.NotNullable(true));
 				comp.Unique(true); // hbm2ddl: Generate a unique constraint in the database
```

Comment:
This test does not need those columns to be nullable, so for avoiding troubles with database not supporting null in unique constraint, setting the columns as not nullable.